locksmith/migrations/20230524112140-add-vat-rate-receipt-base.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+const table = 'ReceiptBases'
+/** @type {import('sequelize-cli').Migration} */
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.addColumn(table, 'vatRatePercentage', {
+      type: Sequelize.INTEGER,
```

Comment:
This is not good. Some VAT rates might be 8.75%

If you want to use `Sequelize.INTEGER` then we need to represent VAT rates as "basis points" like 2000 for 20% or 875 for 8.75%

Reply:
renamed as `vatBasisPointsRate`